incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -509,6 +508,10 @@ HNC propagates `Roles` and `RoleBindings` by default. You can also set any type
 of Kubernetes resource to any of the propagation modes discussed above. To do
 so, you need cluster privileges.
 
+**WARNING: If you start propagating a new object type, HNC _cannot_ check
+whether there are conflicting objects in descendant namespaces, and will
+overwrite them. This will be fixed in HNC v0.6 (see #1076).**
```

Comment:
#1102 may be more specific to this warning, but your call to reference the object overwriting issue (#1076) or the hncconfig update mode issue (#1103) here.

Reply:
Done